code/maps/map_shiny.R

```diff
@@ -1,41 +1,55 @@
-#################################
-# NOTE: run map_prep.R first to create the input datasets county_demo and ppp_census
-#################################
+## open issues ##
```

Comment:
This is just an idea, not a comment on the code. It could be interesting to add a scatter plot to this also, where the axes are the demographics and loan representations selected in the panel; so both map and scatter could update when different variables were selected (apologies if something this is already here; I haven't looked at it live in a while).

Reply:
Exactly what I'm thinking!! I will make some issues for additional features like this.